src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -249,6 +249,12 @@ public class CoinstakeContext
         /// <summary>Indicates that the stake flag is in progress.</summary>
         public const int StakeInProgress = 1;
 
+        /// <summary>
+        /// We don't take coins that are smaller that 0.1 as described in
+        /// <see cref="https://github.com/stratisproject/StratisBitcoinFullNode/issues/1180"/>
```

Comment:
This should be in `<seealso>` section in the summary. Also `s`take.

Reply:
OK will do, I don't understand what you mean by "Also s take" do you just want to replace the verb "take" with "stake" ?

Reply:
We don't take coins that are smaller that 0.1
->
We don't `s`take coins that are smaller tha`n` 0.1

Reply:
if you want... OK